src/server/cluster/cluster_family.h

```diff
@@ -44,6 +44,9 @@ class ClusterFamily {
 
   size_t MigrationsErrorsCount() const ABSL_LOCKS_EXCLUDED(migration_mu_);
 
+  // return number of all keys that were sent
+  size_t MigratedKeysCount() const ABSL_LOCKS_EXCLUDED(migration_mu_);
```

Comment:
style: a function name should be a verb, aka `GetMigratedKeysCount`. the only exception to it is inline variables getters like `int count() { return count_;}`